tests/components/demo/test_water_heater.py

```diff
@@ -22,9 +21,10 @@ def setUp(self):  # pylint: disable=invalid-name
         """Set up things to be run when tests are started."""
         self.hass = get_test_home_assistant()
         self.hass.config.units = IMPERIAL_SYSTEM
-        assert setup_component(
-            self.hass, water_heater.DOMAIN, {"water_heater": {"platform": "demo"}}
-        )
+        assert setup_component(self.hass, water_heater.DOMAIN,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/app/foreign-sub/home-assistant/pullRequest?prid=4636028)